src/utils/loadEnvironmentVariableWithFallback.ts

```diff
@@ -0,0 +1,23 @@
+export const loadEnvironmentVariableWithFallback = ({
```

Comment:
💬 There are other utils we reuse from repo to repo, do we want some utils package that we can share?

Reply:
We could add a GH issue for that and address this outside of this PR